lib/dotinstall/util.py

```diff
@@ -115,6 +115,31 @@ def create_symlink(source: str, target: str) -> None:
     os.symlink(source, target)
 
 
+def concat_files(sources, target, comment_prefix=None):
+    for i, _ in enumerate(sources):
+        sources[i] = os.path.abspath(os.path.expanduser(sources[i]))
+    target = os.path.expanduser(target)
+    _remove_symlink(target)
+    create_dir(os.path.dirname(target))
+    print('Concatenating %s to %s...' % (sources, target))
+    with open(target, 'w') as ft:
+        if comment_prefix:
+            ft.write('%sDO NOT EDIT THIS FILE%s'
+                     % (comment_prefix, os.linesep))
+            ft.write('%sThis file was concatenated by dotfiles utility.%s'
+                     % (comment_prefix, os.linesep))
+            ft.write('%sAll changes in this file will not be retained.%s%s'
+                     % (comment_prefix, os.linesep, os.linesep))
```

Comment:
…and here use
```python3
print(comment_fmt.format('your message…'), file=handle)
```
that way you don't need os.linesep, `print`  will take care of it for you.
To print a blank line to a file, no param is needed:
```python3
print(file=handle)
```

Reply:
Done.